docs/README.md

```diff
@@ -2544,6 +2615,32 @@ source <path-to-your-bash-file> && boost deploy -e production
 
 Now just let the magic happen, Booster will create everything for you and give you back your app ready to use URL. 🚀
 
+##### Kubernetes provider
+
+To configure Kubernetes as a provider you need to meet certain prerequisites:
+
+- Config the Kubernetes cluster beforehand in a cloud provider or on-premises.
+- Install **kubectl**.
+- Install [Helm](https://helm.sh).
+- Check `@boostercloud/framework-provider-kubernetes` is listed in your app `package.json` dependencies.
+- Check `@boostercloud/framework-provider-kubernetes-infrastructure` is listed in your app `package.json` devDependencies.
+- Check both dependencies are installed, otherwise use `npm install` in the root of your project.
+
+At this moment you should be able to access your Kuberbetes cluster from **kubectl** command.
+
+Now go to your `config.ts` file, import the kubernetes provider library and set up your app environment.
+
+```typescript
+import { Booster } from '@boostercloud/framework-core'
+import { BoosterConfig } from '@boostercloud/framework-types'
```

Comment:
we have a custom configuration for the K8s provider so the import is this one: `import { BoosterK8sConfiguration } from '@boostercloud/framework-provider-kubernetes-infrastructure'`

Reply:
ok